CHANGELOG.md

```diff
@@ -152,6 +154,7 @@ Changed:
 
 - `geopandas` >=0.13 &rarr; >=0.14
 - `pandas` >=1.5,<2.0 &rarr; >=2.1
+- `salib` >=1.3.0 &rarr; >=1.4.7
```

Comment:
@emanuel-schmid : why would we indicate two different larger than versions (as for geopandas)?

Reply:
This is about dependency version constraint changes. They start from the given minimal version of the last release (left hand side) and end at the given minimal version of this release (right hand side).
However: the whole version change section is eventually done by a script I wrote. Don't bother, it will be overwritten!